src/main/java/de/qabel/desktop/ui/accounting/item/AccountingItemController.java

```diff
@@ -87,6 +94,16 @@ public void edit(ActionEvent actionEvent) {
         result.ifPresent(this::setAlias);
     }
 
+    private void initializeMenu() {
+        menuqrView = new MenuQRView();
+        menuqrView.getView(layoutWindow.getChildren()::add);
```

Comment:
It's added to the layoutWindow but never removed. If you add it in `openMenuQR`, you could remove it on close / hide